src/components/Login/Login.jsx

```diff
@@ -0,0 +1,110 @@
+import React, {useState} from 'react';
+import {Redirect} from 'react-router-dom';
+import {connect} from 'react-redux';
+
+import {makeStyles} from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+import {ActionTypes} from '../../constants/Actions';
+import './Login.css';
+
+const useStyles = makeStyles(theme => ({
```

Comment:
We can use both styles, obviously 1 at a time.
I was just showcasing that we can do it both ways as it was an example project.
The good approach would be to move everything in the css.
Or if one wishes to go with the jss approach, instead of defining style in every file, one can maintain a common file which can have centralized styles.